### DISCUSSION TITLE
Unknown Feature

### DISCUSSION THREAD

**Discussion by Newer1107** [prasanthrangan/hyprdots#336] [Q&A] [answered]
What is this? What is activating? I don't seem to find any kind of documentation for this.

https://github.com/prasanthrangan/hyprdots/assets/116626403/ab7c6a13-8e04-4923-b1cf-c6d80507047b

**Comment by T-Crypt** [comment, marked as answer]
That is the idle_inhibitor. 

It inhibits idle mode so you can watch videos without worrying about swaylock popping up.
